rfcs/45-stageless.md

```diff
@@ -62,41 +88,43 @@ enum GameState {
 }
 
 #[derive(SystemLabel)]
-enum GameSet {
+enum MySystemSets {
     Menu,
     SubMenu,
 }
 
+#[derive(SystemLabel)]
+enum MyLabels {
+    FlushMenu,
+}
+
 fn main() {
     App::new()
         /* ... */
-        // helper function that registers state machine resource
-        // and adds OnEnter and OnExit sets for its states
-        .add_states([GameState::Playing, GameState::Paused])
         .add_set(
-            // systems and sets have the same API for scheduling
-            GameSet::Menu
-                // sets can go inside other sets
+            // use the same builder API for scheduling systems and system sets
+            MySystemSets::Menu
+                // put sets in other sets
                 .in_set(CoreSet::Update)
-                // sets can have conditions, these must be true for the systems inside to run
+                // give sets conditions (if one fails, all systems in the set will be skipped)
                 .run_if(state_equals(GameState::Paused))
         )
         .add_system(
             some_system
-                .in_set(GameSet::Menu)
-                // systems and sets can have multiple conditions (they must all be true)
+                .in_set(MySystemSets::Menu)
+                // things can have multiple conditions! (they must all return true)
+                // while being in a set! (there are no conflicts)

```

Comment:
```suggestion
                // systems and sets can have multiple conditions, which must all return true.
```

Reply:
So here, I was trying to highlight that something can have multiple conditions, while being in a system set that has its own condition. Neither of these things are possible in current Bevy.

Reply:
Can you at least rework the second line? It doesn't really make sense grammatically

Reply:
Sure, I can do that.